packages/edit-site/src/components/page-switcher/index.js

```diff
@@ -33,8 +33,13 @@ export default function PageSwitcher( {
 						const path = getPathFromLink( _page.link );
 						return {
 							label: _page.title.rendered,
+							type: 'page',
+							slug: _page.slug,
```

Comment:
Is this the template slug?

Reply:
The page slug.

Reply:
🙃 I realized that's what the variable is called, but was originally wondering if it had to do with the page page instead of the page cpt

Reply:
Well, the page page's slug is the same as the page CPT slug in this case. It's what we need to generate its specific template slug.

Reply:
Ah, that makes sense